contracts/payloads/IGP93/PayloadIGP93.sol

```diff
@@ -420,12 +422,12 @@ contract PayloadIGP93 is PayloadIGPMain {
         {
             FluidLiquidityAdminStructs.UserSupplyConfig[] memory configs_ = new FluidLiquidityAdminStructs.UserSupplyConfig[](1);
             configs_[0] = FluidLiquidityAdminStructs.UserSupplyConfig({
-                user: address(0), // Protocol address or do we put Lending Factory = 0x54B91A0D94cb471F37f949c60F7Fa7935b551D03
-                token: 0x90551c1795392094FE6D29B758EcCD233cFAa260, // fToken_fWETH
+                user: getFTokenAddress(ETH_ADDRESS), // fToken_fWETH
```

Comment:
special case here, for getting fToken address we must use WETH address 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 

but for token at liquidity layer stay with ETH_ADDRESS